lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -816,6 +816,12 @@ RawMemManagerCore::getGlobalVariableInitValue(const GlobalVariable &gv) {
 bool RawMemManagerCore::isPtrTyVal(Expr e) {
   return (!e || !strct::isStructVal(e));
 }
+RawMemManagerCore::MemValTy
+RawMemManagerCore::setMemory(unsigned int val) const {
+  assert(llvm::Log2_64(val) + 1 <= wordSizeInBits());
```

Comment:
there should be a warning printed if assertion fails. This can be hard to trace in non-debug mode